tikv/raftstore/peer_state.go

```diff
@@ -16,6 +16,9 @@ type peerState struct {
 	handle unsafe.Pointer
 	peer   *peerFsm
 	apply  *applyFsm
+
+	// redoIdx is the raft log index starts redo for lockstore.
+	redoIdx uint64
```

Comment:
What's the different between redoIndex and redoIdx?

Reply:
It's a redudant field, not used, I'll fix it.